doc/articles/get-started-vscode.md

```diff
@@ -72,7 +72,6 @@ This will create a solution that only contains the WebAssembly and Skia+GTK plat
 ## Run and Debug application
 
 ### WebAssembly
-1. In the debugger section of the Code activity bar, select `Debug (Chrome, WebAssembly)`
```

Comment:
Did you see this option as the default debugger target in your VS Code?

Reply:
I see. Then maybe the text is not the right one, but even if it's the default, I'd rather see developers understand that this is the debugger target to use, particularly if they changed it because of Skia targets.

Maybe:
```
Ensure that in the debugger section of the Code activity bar, select `Debug (Chrome, WebAssembly)`
```